Library/Homebrew/development_tools.rb

```diff
@@ -54,13 +53,6 @@ def gcc_42_build_version
     end
     alias_method :gcc_build_version, :gcc_42_build_version
 
-    def llvm_build_version
-      @llvm_build_version ||=
```

Comment:
Now that this is gone, resetting this instance variable in `clear_version_cache` further down doesn't make any sense.